src/cli/util.js

```diff
@@ -37,9 +37,12 @@ function getOptions(argv) {
   );
 }
 
-function dashifyObject(object) {
+function cliifyOptions(object) {
   return Object.keys(object || {}).reduce((output, key) => {
-    output[dashify(key)] = object[key];
+    const apiOption = constant.apiDetailedOptionMap[key];
+    const cliKey = apiOption ? apiOption.name : key;
```

Comment:
Need to map "plugins" to "plugin", else it always gets clobbered (even if `--plugin` isn't specified on CLI)